android/src/main/kotlin/com/apptreesoftware/flutterwebview/FlutterWebViewPlugin.kt

```diff
@@ -67,7 +67,7 @@ class FlutterWebViewPlugin(val activity: Activity) : MethodCallHandler {
                 //val tintColor = call.argument<String?>("tint")
                 //val barColor = call.argument<String?>("barTint")
 
-                val url = call.argument<String>("url")
+                val url = call.argument<String>("url") ?: ""
```

Comment:
Rather than defaulting to an empty string, which is not valid for any of these method calls, it would be better to throw an exception here. Something like

```kotlin 
val url = call.argument<String>("url") ?: throw IllegalArgumentException("url must not be null").
```

I believe this would apply for all changes you've made in this PR.